src/components/Votes.jsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from "react";
+import { BiLike, BiDislike } from "react-icons/bi";
+import { patchArticleVotes } from "../utils/api";
+
+function Votes({ article, setArticle, voteCount, setVoteCount }) {
+  const [err, setErr] = useState(null);
+
+  function handleLike(event) {
+    event.preventDefault();
+    setVoteCount((voteCount) => voteCount + 1);
+    setErr(null);
+    patchArticleVotes(article.article_id, 1)
+      .catch((err) => {
+        setVoteCount((voteCount) => voteCount - 1);
+        setErr("Something went wrong!");
+      })
+      .then((updatedArticle) => {
+        setArticle(updatedArticle);
+      });
```

Comment:
This chain isn't quite right: 
1. in terms of ordering, the catch should be at the end of the chain. 
2. In terms of what setArticle is doing. 
I think updatedArticle would most likely be undefined, which is why it crashes - do you need this .then bit? Not sure what its meant to do. 

At the moment, it crashes the whole app if you try and click the button whilst offline.